jaeger_client/thrift.py

```diff
@@ -26,6 +26,24 @@
     long = int
 
 
+def id_to_low(big_id):
+    """
+    :param big_id: id in integer
+    :return: Returns the right most 64 bits of big_id
+    """
+    if big_id is not None:
+        return big_id & (_max_unsigned_id - 1)
```

Comment:
does Python assume None return value if there's no explicit return at the end?

Reply:
Yes. Functions in Python will return `None` if we don't return anything explicitly